README.md

```diff
@@ -1,2 +1,61 @@
+
+Once MLOps comes back to you with a shiny new Outerbounds account and a perimeter for your team, you can sign in [here](https://docs.google.com/document/d/12S06Q-9xh6YGkyBpi6iGWGazYDivxl2fT4KaHtbrzV4/edit).
+
+Go to [this page right here](https://ui.desertowl.obp.outerbounds.com/dashboard/configure?location=local). You’ll see some instructions. You have already installed Outerbounds, so you get to skip that one. 
+Look for a dropdown that says “perimeter.” You want to make sure that, if your data has more restricted access than All-Mozilla, you have checked the perimeter name that you received from MLOps. If your data has all-Mozilla access, you can check “default” here.
+
+You should see an instruction to run a command line command that starts with `outerbounds configure`. Copy that whole thing and run it on your command line. What this does, is it makes a `config.json` file located at `~/.metaflowconfig` on your local machine. This is the default configuration for metaflow on your machine now, and unless told otherwise, metaflow flows on your machine will now always run on Outerbounds. 
+
+### Once you have done the above, you're ready to use the template in this repository.
+
+1. Install the requirements listed in the `requirements.txt` in the `templates` directory of this repo, and also add those to your project requirements files.
+2. Copy the `template_flow.py` file from ths repo into _your_ repo. From there, you can run `python template_flow.py run --offline True`.
+
+You will see your metaflow flow kick off with such proclamations as "The graph looks good!" and "Pylint is happy!", and then, [you can go here](https://ui.desertowl.obp.outerbounds.com/dashboard/runs) (to the “Runs” tab in the Outerbounds UI) to track the progress of the flow, see stderr and stdout, et cetera. 
+
+Eventually you should see the flow finish with "Task finished successfully." Once you see that, you know you've got a working minimal flow: you're now ready to start putting your model training code _into_ this flow to run it on the Outerbounds instance. Comments _inside_ the template file should help you understand where to put that code, and demonstrate some of the tools Metaflow makes available to you.
+
+ ## Next: A Fancy Dashboard
+
+An admin from Mozilla’s MLOps team needs to set you up with a Weights and Biases User account, and also a team if your team doesn’t have that yet, on Weights and Biases. Reach out to #mlops in Slack for help with this.
+
+MLOps needs to know:
+1. If you will need a service account for your Weights and Biases team—say, to set up CI or something. Seats cost significant amounts of money, so if you need extra accounts “just in case,” no you don’t. But if you are, for example, already set up with CI for your model and you want that on Weights and Biases, we can get you a service account for that.
+
+Once you have the account, you're in luck; the `template_flow` already integrates with Weights and Biases for you. 
```

Comment:
So this is an example of the sort of thing that I think is a conversation rather than a bunch of commit suggestions made without additional comment. 

It feels like you don't think there's a good reason I did it this way, other than maybe ignorance to convention, which feels like a lack of trust in me. I would have expected this to be a question. "Hey, I noticed that the whole flow takes a colloquial tone. That's unconventional. Why did you make that choice?"

Reply:
> It feels like you don't think there's a good reason I did it this way,

I never said or implied that. Reviews are conversations and all the changes suggested here were made in the same spirit of the comments on the original document. The dropped text doesn't add much, _in my opinion of a non-native English speaker_, to the content: doesn't make me more able to reason about the flow and it contributes to the overall length of this document. I believe this is too long of a readme to explain how to use one template file.

Reply:
So, the reason I did it this way is that I foresee a miscommunication with README users in which they think "The `template_flow` already integrates with Weights and Biases for you" means they don't need, or aren't blocked on, the account creation steps explained above. I think that's particularly true now that we have moved the account creation text to Confluence, where likely have not seen it by the time they get here. 

I believe taking a little extra time to mitigate risk of miscommunication is usually worth it. It been my humble experience that I have lost way more time to making up for incomplete, assumptive documentation than I have to reading a _slightly_ longer piece ahead of using the code it describes.